sda-commons-server-key-mgmt/src/main/java/org/sdase/commons/keymgmt/KeyMgmtBundle.java

```diff
@@ -154,9 +171,22 @@ public FinalBuilder<C> withFailStrategy(FailStrategy strategy) {
       return this;
     }
 
+    @Override
+    public FinalBuilder<C> withApiPlaceholder(String apiPlaceholder) {
```

Comment:
We had explicitly decided in our last meeting that the placeholder value must not be configurable. What we want instead is one static value that is predefined by the framework and that must not be modifiable.
I just talked to Andre about this and he decided that we need three new mapping strategies. The names of these three new mapping strategies are as follows:
PLACEHOLDER_BIDIRECTIONAL   -> uses a placeholder for both directions
PLACEHOLDER_IMPL_TO_API  -> uses a placeholder when mapping from impl to api. For api to impl, an exception is thrown for unmappable keys.
 PLACEHOLDER_API_ TO_IMPL -> uses a placeholder when mapping from api to impl. For impl to api, an exception is thrown for unmappable keys.

If anything is not clear, please ask.